### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by badhrinathpa**
Attach Rej handling + sec mode failure attach rej handling + Service req state issue fix + service rej encoding

### CODE REVIEW

File: src/mme-app/handlers/attach_secmoderesp.c [hunk 1]
```diff
@@ -172,6 +172,29 @@ post_to_next(char *buf)
                                   Send Initial Ctx setup request");
         return send_init_ctx_setup_req(secmode_resp->ue_idx);
     }
+    else if ((ATTACH_PROC == ue_entry->ue_curr_proc)
+             &&(STAGE4_FAIL == ue_entry->ue_state))
+    {
+        log_msg(LOG_DEBUG, "Sec mode cmd failed in attach req proc. \
+                                 Send attach reject and clear context");
+        log_msg(LOG_INFO, "Sending Attach Reject\n");
```
Comment (thakurajayL) [inline src/mme-app/handlers/attach_secmoderesp.c]: Looks extra log. If possible print UE identity IMSI

File: src/mme-app/handlers/service_req.c [hunk 2]
```diff
@@ -206,15 +207,23 @@ post_to_next()
 static int
 post_service_reject()
 {
-    struct commonRej_info s1ap_rej;
-	log_msg(LOG_INFO, "Sending Service Rej \n");
-	struct service_req_Q_msg *service_req =
-				(struct service_req_Q_msg *) buf;
-    s1ap_rej.IE_type = S1AP_SERVICE_REJECT;
+    log_msg(LOG_INFO, "Sending Service Rej \n");
```
Comment (thakurajayL) [inline src/mme-app/handlers/service_req.c]: Lets add available User identifier in the logs.

**Review by thakurajayL — APPROVED** [review]

File: src/mme-app/handlers/attach_secmoderesp.c (hunk 1 above)
Comment (badhrinathpa) [reply src/mme-app/handlers/attach_secmoderesp.c]: done

File: src/mme-app/handlers/service_req.c (hunk 2 above)
Comment (badhrinathpa) [reply src/mme-app/handlers/service_req.c]: done. added mTMSI

**Pull request merged**